vscode-antimony/src/server/stibium/test/test_grammar.py

```diff
@@ -0,0 +1,384 @@
+import os
+import sys
+
+ROOT = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.append(ROOT)
+
+from bioservices_server.utils import sb_position
+from typing import Tuple
+from stibium.analysis import AntTreeAnalyzer
+from stibium.ant_types import Annotation, ArithmeticExpr, Assignment, DeclItem, DeclModifiers, Declaration, FileNode, InComp, Keyword, Name, NameMaybeIn, Number, Operator, Reaction, ReactionName, SimpleStmt, Species, SpeciesList, StringLiteral, VarName
+from stibium.api import AntFile
+from stibium.parse import AntimonyParser
+from stibium.types import IncompatibleType, IssueSeverity, ObscuredValue, SrcPosition, SrcRange, UnexpectedEOFIssue, UnexpectedNewlineIssue, UnexpectedTokenIssue
+
+from pygls.workspace import Document
+
+import pytest
+
+parser = AntimonyParser()
+directory = os.path.join(os.path.dirname(os.path.abspath(__file__)), "test-data")
+
+# some of the following test data are from Lucian Smith, https://github.com/sys-bio/antimony/tree/develop/src/test/test-data
+
+    
+
+@pytest.mark.parametrize('code,expected_parse_tree_str', [
+    ('# this is a comment', "Tree('root', [])"),
+    ('// this is another comment', "Tree('root', [])"),
+    ('/* one more cooment \n new line here */', "Tree('root', [])"),
+])
+def test_comment(code, expected_parse_tree_str):
+    '''
+    test comments
+    '''
+    antfile = AntFile('', code)
+    assert len(antfile.get_issues()) == 0,\
+        f"this test has no dependencies"
+    tree = parser.get_parse_tree_str(code)
+    assert str(tree) == expected_parse_tree_str,\
+        f"this test has no dependencies"
+        
+        
+@pytest.mark.parametrize('file_name,expected_parse_tree_str', [
+    ('reaction001',"Tree('root', [Tree('simple_stmt', [None, Token('NEWLINE', '\\n')]), Tree('simple_stmt', [Tree('reaction', [Tree('reaction_name', [Tree('namemaybein', [Tree('var_name', [None, Token('NAME', '_J0')]), None]), Token('COLON', ':')]), Tree('species_list', [Tree('species', [None, None, Token('NAME', 'S1')])]), Token('ARROW', '->'), None, Token('SEMICOLON', ';'), Tree('product', [Tree('var_name', [None, Token('NAME', 'default_compartment')]), Token('STAR', '*'), Tree('var_name', [None, Token('NAME', 'S1')])]), None]), Token('SEMICOLON', ';')]), Tree('simple_stmt', [None, Token('NEWLINE', '\\n')]), Tree('simple_stmt', [Tree('assignment', [Tree('namemaybein', [Tree('var_name', [None, Token('NAME', 'default_compartment')]), None]), Token('EQUAL', '='), Token('NUMBER', '1')]), Token('SEMICOLON', ';')]), Tree('simple_stmt', [None, Token('NEWLINE', '')])])"),
```

Comment:
is there a way to save all of these to a file instead of hard-coding the string here? very hard to read.